tools/generate-imports.mjs

```diff
@@ -152,13 +152,18 @@ async function generateHash() {
       .map((file) => file.name);
 
     for (const manager of managers) {
-      const hash = await getManagerHash(manager);
+      const hash = await getManagerHash(
+        manager === 'custom' ? 'custom/regex' : manager
```

Comment:
🤔 looks wrong to me. didn't we concluded to name it `custom.regec`?

Reply:
This PR doesn't do any logic change only refactoring